tests/CompilerModuleOverrideTest.php

```
@@ -0,0 +1,145 @@
+        $consumer = $injector->getInstance(FakeMultiBindingConsumer::class);
+        $this->assertInstanceOf(Map::class, $consumer->engines);
+        $this->assertArrayHasKey('test', $consumer->engines);
+    }
+}
+
+/**
+ * Fake custom injector for testing override behavior
+ */
+class FakeCustomInjector implements InjectorInterface
+{
+    public function getInstance($interface, $name = '')
```

PHPMD (2.15.0)

Avoid unused parameters such as '$interface'. (undefined)

(UnusedFormalParameter)